lib/assets/javascripts/jasmine-runner.js

```diff
@@ -90,6 +90,10 @@
 
   var address = args[1];
   console.log('Running: ' + address);
+  if(phantom.version.major===2){
```

Comment:
Could you please format this in standardjs style?

```js
if (phantom.version.major === 2) {
  address += '\n'
}
```

Reply:
done